android/src/main/java/com/revenuecat/purchases/hybridcommon/common.kt

```diff
@@ -322,14 +288,21 @@ fun configure(
     apiKey: String,
     appUserID: String?,
     observerMode: Boolean?,
-    platformInfo: PlatformInfo
+    platformInfo: PlatformInfo,
+    store: Store = Store.PLAY_STORE,
+    dangerousSettings: DangerousSettings = DangerousSettings(autoSyncPurchases = true)
 ) {
     Purchases.platformInfo = platformInfo
+    val builder =
+        PurchasesConfiguration.Builder(context, apiKey)
+            .appUserID(appUserID)
+            .store(store)
+            .dangerousSettings(dangerousSettings)
     if (observerMode != null) {
-        Purchases.configure(context, apiKey, appUserID, observerMode)
-    } else {
-        Purchases.configure(context, apiKey, appUserID)
+        builder.observerMode(observerMode)
     }
+
+    Purchases.configure(builder.build())
 }
 
 fun getPaymentDiscount() : ErrorContainer {
```

Comment:
this should be renamed to getPromotionalOffer, but it can also be done as a separate PR